docs/csharp/whats-new/csharp-7-3.md

```diff
@@ -0,0 +1,184 @@
+---
+title: What's new in C# 7.3
+description: An overview of new features in C# 7.3
+ms.date: 05/16/2018
+---
+# What's new in C# 7.3
+
+There are two main themes to the C# 7.3 release. One theme provides features that enable safe code to be as performant as unsafe code. The second theme provides incremental improvements to existing features. In addition, new compiler options were added in this release.
+
+The following new features support the theme of better performance for safe code:
+
+- You can access fixed fields without pinning.
+- You can reassign `ref` local variables.
+- You can use initializers on `stackalloc` arrays.
+- You can use `fixed` statements with any type that supports a pattern.
+- You can use additional generic constraints.
+
+The following enhancements were made to existing features:
+
+- You can test `==` and `!=` with tuple types.
+- You can use expression variables in more locations.
+- You may attach attributes to the backing field of auto-implemented properties.
+- Method resolution when arguments differ by `in` has been improved.
+- Overload resolution now has fewer ambiguous cases.
+
+The new compiler options are:
+
+- `-publicsign` to enable OSS signing of assemblies.
+- `-pathmap` to provide a mapping for source directories.
+
+The remainder of this article provides details and links to learn more about each of the improvements.
+
+## Enabling more performant safe code
+
+You should be able to write C# code safely that performs as well as unsafe code. Safe code avoids classes of errors, such as buffer overruns, stray pointers, and other memory access errors. These new features expand the capabilities of verifiable safe code. Strive to write more of your code using safe constructs. These features make that easier.
+
+### Indexing `fixed` fields does not require pinning
+
+Consider this struct:
+
+```csharp
+unsafe struct S
+{
+    public fixed int myFixedField[10];
+}
+```
+
+In earlier versions of C#, you needed to pin a variable to access one of the integers that are part of `myFixedField`. Now, the following code compiles:
+
+```csharp
+var s = new S();
+int p = s.myFixedField[5];
+```
+
+The variable `p` doesn't need to be pinned. Note that you still need an `unsafe` context.
+
+You can learn more in the article on the [`fixed` statement](../language-reference/keywords/fixed-statement.md).
```

Comment:
You can learn more -> The boilerplate sentence for this kind of sentence is usually "For more information, see..." or "For more information about x, see ...". I think it helps to localize as well since it's a standard sentence.

(several times on this topic)

Reply:
I reworded all of them.